core/new-gui/src/app/workspace/service/operator-metadata/operator-metadata.service.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { TestBed, inject } from '@angular/core/testing';
+
+import { HttpClient } from '@angular/common/http';
+import { OperatorMetadataService } from './operator-metadata.service';
+
+import { Observable } from 'rxjs/Observable';
+
+import '../../../common/rxjs-operators';
+
+
+class StubHttpClient {
+  constructor() { }
+
+  public get(url: string): Observable<any> {
+    return Observable.of('test response');
+  }
+}
+
+describe('OperatorMetadataService', () => {
+
+  let service: OperatorMetadataService;
```

Comment:
Shouldn't this be StubOperatorMetadataService?
However, in order to make sure that when server's operator schema changes, the front end's tests fail, we should contact the actual backend and not just mock it.